src/watcher.js

```diff
@@ -341,7 +378,8 @@ class Watcher extends SafeEventEmitter {
             if (currentChannel.id === channel.id) return;
             channel = currentChannel;
 
-            api.get(`channels/${channel.name}`)
+            api
```

Comment:
it looks like you have some autoformatter on that is causing unrelated changes. please disable that and undo these unrelated changes.

Reply:
yeah, i've prettier autoformatting that for me, i'll revert these